x/stake/wire.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/wire"
 )
 
-// XXX complete
+// TODO complete when go-amino is ported
```

Comment:
"Aminoification" is done AFAIK - are we waiting for some part still?

Reply:
addressed